New Syndicate Business Missions (#7245)
endless-sky/endless-sky: data/human/human missions.txt

			label end
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

mission "Syndicate Business 1"
	name "Rescue Kidnapped Man in <waypoints>"
	description "A mysterious man has been kidnapped by a heavy pirate warship in <waypoints>. Destroy the ship to rescue him."
	minor
	to offer
		random < 40
		credits > 5000000
	source 
		government Syndicate
		attributes "core"
	waypoint "Moktar"
	destination "Sunracer"
	on offer
		conversation

> 			`Returning to your ship after an uneventful dinner at the spaceport, your ship's communication screen blares an alarm as the words "EMERGENCY DISTRESS SIGNAL" flash in red. A terrified-looking man in a suit appears on the screen. "I don't have much time - I've been kidnapped. If you're receiving this message, I need you to destroy the ship. I've managed to barricade myself in an escape pod."` 

Reviewer: How is the player getting this signal? How is the man sending out the signal? Why doesn't anybody else get the signal? How does he manage to hold out for over a day?

> 			`You hear banging and muffled shouting over the intercom. "I'm a Syndicate employee - I can pay you! Help me captain, you're my only h-" The message cuts out.`

Reviewer: ```suggestion
			`	You hear banging and muffled shouting over the intercom. "I'm a Syndicate employee - I can pay you! Help me captain, you're my only h-" The message cuts out.`
```

			choice
				`	(Assist the mysterious man.)`
					goto assist
				`	(Don't assist the man.)`
					decline
			label assist

> 			`	You note that the distress signal came from a warship in <waypoints> and wonder why a Syndicate employee would be kidnapped by pirates. Anyway, time to get ready for a fight.`

Reviewer: ```suggestion
			`	You note that the distress signal came from a warship in <waypoints> and wonder why a Syndicate employee would be kidnapped by pirates.`
```

				accept
	on enter "Moktar"
		"reputation: Independent (Killable)" = -1000
		conversation
			`When you enter <waypoints>, your sensors detect a pirate Vanguard, but no sign of any escape pod. Your ship is immediately hailed, and the pirate opens communication.`

> 			`	The voice of the captain of the Golden Jewel shouts out towards you. "You were unwise to get involved in our business. Hand over that pompous salesman Howard and I'll let you live."`

Author: This one I don't think will work given the npc hasn't spawned yet. I'll playtest before clearing though

			choice
				`	"Uhhh... hand over who?"`
					goto who
				`	"I don't negotiate with pirates!"`
					goto negotiate
				`	"You can take him, but I don't know where he is."`
					goto who
			label who
			`	"Bah, you lying buffoon! I'll find him in your ship's wreckage!"`
				goto power
			label negotiate
			`	"Aye, then I won't negotiate with you!"`
				goto power
			label power
			`	The Vanguard powers up its weapons.`
	npc kill
		government "Independent (Killable)"
		personality heroic vindictive staying
			confusion 20
		system "Moktar"
		ship "Vanguard" "Golden Jewel"		
		conversation 

> 			`The <npc> has been destroyed. You receive a hail from a small escape pod. "Thank you, Captain, you've saved my life. I'm Howard. Meet me on <destination> to receive your reward."`

Reviewer: But how does he get to Sunracer?

Reviewer 2: I think we could say you collect the escape pod? Or maybe make you board him?

			choice
				`	"Sounds good."`

> 					accept

Reviewer: Style change.
```suggestion
					decline
```

				`	"What's the deal with you and that pirate?"`
					goto piratedeal

> 			label piratedeal

Author: I have a question on this set of edits. I see the redundant goto/labels as making the missions more readable for humans, but is it best practice to remove them?

> 			label piratedeal

Reviewer 3: Generally best practice is to avoid unnecessary bloat, but I personally don't think most of these are, like, egregious enough to prevent merging.

The only one I think is bad enough that it should definitely be removed is this one here, since the label doesn't seem to be used at all otherwise.

> 			label piratedeal

Reviewer 2: If you want to make it more human readable you should probably add clear comments instead

> 			`"That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to go to <destination>.`

Reviewer: ```suggestion
			`	"That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to go to <planet>.`
```

	on complete
		"reputation: Independent (Killable)" = 10
		conversation

> 			`When you land on <destination>, Howard is waiting for you at the spaceport. "Thank you, truly. I'm Howard Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits.`

Reviewer: ```suggestion
			`When you land on <planet>, Howard is waiting for you at the spaceport. "Thank you, truly. I'm Howard Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits.`
```

> 			`	However, I see you're clearly a captain of means. I'd have to pull a few strings, but what if instead I get you into a private placement of Maralex stock? To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits."`

Reviewer: ```suggestion
			`	However, I see you're clearly a captain of means. I'd have to pull a few strings, but I could get you into a private placement of Maralex stock. To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits."`
```

					goto maralex
				`	"Private placement? Does that mean I own part of the Syndicate?"`
					goto placement
				`	"No thanks. I'll take the money, please."`
					goto payup
			label maralex
			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We design, manufacture, and integrate innovative micro-control solutions for high-performance commercial and military air and space vehicles. Succinctly, we make the airframes of small planes and spacecraft. Would you like to invest?"`
			choice
				`	"Sure, I'll invest. What do I need to do?"`
					goto invest
				`	"Nope, I'll stick with the 90k."`	
					goto payup
			label placement
			`	"Almost. You will own a fraction of a partially owned subsidiary of Syndicated Systems - Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest in us?"`
			choice
				`	"Sure, I'll invest. What do I need to do?"`
					goto invest
				`	"Nope, I'll stick with the 90k."`
					goto payup
			label invest

> 			`	"Excellent. To purchase 1 share of Maralex Controls I will need 1 million credits. Then you sign a few documents, send over the payment, and take care of a few other administrative tasks. I'll need to get approval for the stock sale and set up the paperwork, so it won't be ready today, but feel free to come back to my office here anytime to execute the purchase. Remember, you'll need to have at least 1 million credits for the stock purchase."`

Reviewer: You haven't explicitly mentioned to check the spaceport.

> 				accept

Reviewer: ```suggestion
				decline
```

			label payup

> 			`	"Erhm, yes, of course. Here you are, my savings of ninety-one thousand, nine hundred and fifty one credits. I remain in your debt, and if you still want to invest in us, I will be here."`	

Reviewer: The player will still get the following mission after taking this choice.

			action
				payment 91951

> 		log `Howard Vendeur from Maralex Controls offered to let you buy stock for 1 million credits on Sunracer.`

Reviewer: Unnecessary; the following mission already tells the player this information.
```suggestion
```

mission "Syndicate Business 2"
	name "Purchase Stock on <planet>"
	description "Howard from Maralex Controls is letting you buy a share of the company for 1 million credits"

> 	priority

Reviewer: ```suggestion
```

	passengers 1
	source "Sunracer"
	destination "Hephaestus"
	to offer
		has "Syndicate Business 1: done"
		credits > 1000000
	on offer
		conversation
			`Howard once again thanks you profusely for saving his life and excitedly tells you what an exclusive offer you've received to be able to purchase a share in Maralex Controls.`
			`	"To reiterate, to purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits. Do you accept these terms?"`
			choice
				`	"Sure, here's the 1 million credits."`
					goto doit
				`	"Nope, I won't become an investor."`	
					decline
			label doit

> 			`After you've signed numerous dotted lines, Howard says,"Alright, we have your payment and preliminary signatures. All that's left is to submit this first set of paperwork to our headquarters and then go to <destination> to sign the final documents and conduct a biometric scan at our secure facility. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`

Reviewer: ```suggestion
			`After you've signed numerous dotted lines, Howard says, "Alright, we have your payment and preliminary signatures. All that's left is to submit this first set of paperwork to our headquarters and then go to <destination> to sign the final documents and conduct a biometric scan at our secure facility. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`
```

				accept
	on accept
		payment -1000000
	on complete
		conversation
			`You land on <destination> and follow Howard's instructions to the biometric scan center. You hand in the first set of paperwork you filled, and after scanning your fingerprint, eyes, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign." He places two sets of documents on the table and taps the set on the left.`

> 			`	"New regulations dictate that you have a choice in investing in our full spectrum of activities, or use the alleged 'ethical investment' option to avoid funding more ambiguous activities such as strip mining, clearing endangered habitats, and doing business on worlds with alleged human rights violations."`

Author: I prefer your correction

			`	He then taps the documents on the right and says, "The second option would of course reduce your total returns by roughly 40%, resulting in a daily dividend of 300 credits. This is your final decision and set of paperwork. So, would you like to invest in the full investment suite or in the ethical investment option?"`
			choice

> 				`	"Full investment suite." Sign the left set of documents. (500 credits daily)`

Reviewer 4: ```suggestion
				`	"Full investment suite." (500 credits daily)`
```

					goto high
				`	"I'd prefer the ethical investment option." (300 credits daily)`	
					goto low

> 			label high

Reviewer: This leads directly into the low profit branch.

> 				`	"I'd prefer the ethical investment option." Sign the right set of documents. (300 credits daily)`	

Reviewer 4: ```suggestion
				`	"I'd prefer the ethical investment option." (300 credits daily)`	
```

			action
				"salary: Syndicate Business" = 500
			label low
			action
				"salary: Syndicate Business" = 300

> 			`"Excellent. As a part owner of Maralex Controls, you will now receive a daily dividend. I have some work to attend to while I'm at our headquarters, but it was a pleasure doing business with you, Captain."`

Reviewer: ```suggestion
			`	"Excellent. As a part owner of Maralex Controls, you will now receive a daily dividend. I have some work to attend to while I'm at our headquarters, but it was a pleasure doing business with you, Captain."`
```

			label end
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

> mission "Syndicate Business 1, start"

Reviewer 4: ```suggestion
mission "Syndicate Business 1"
```

	name "Rescue Kidnapped Man on Moktar"

> 	description "The mysterious man has been kidnapped by a heavy pirate warship in Moktar. Destroy the ship to rescue him"

Reviewer 4: ```suggestion
	description "A mysterious man has been kidnapped by a heavy pirate warship in Moktar. Destroy the ship to rescue him"
```

> 	description "The mysterious man has been kidnapped by a heavy pirate warship in Moktar. Destroy the ship to rescue him"

Reviewer 5: So apparently Github decided to not notify me about the recent changes. Some of these might already be fixed.

	minor
	to offer
		random < 40
		credits > 5000000
	source 
		government Syndicate
		attributes "core"
	waypoint "Moktar"
	destination "Sunracer"
	on offer
		conversation

> 			`Returning to your ship after an uneventful dinner at the spaceport, your ship's communication screen blares an alarm as the words EMERGENCY DISTRESS SIGNAL flash in red. A terrified looking man in a suit appears on the screen. 'I don't have much time — I'm kidnapped. If you're receiving this message, I need you to destroy the ship. I've managed to barricade myself in an escape shuttle."` 

Reviewer 4: ```suggestion
			`Returning to your ship after an uneventful dinner at the spaceport, your ship's communication screen blares an alarm as the words EMERGENCY DISTRESS SIGNAL flash in red. A terrified looking man in a suit appears on the screen. 'I don't have much time — I'm kidnapped. If you're receiving this message, I need you to destroy the ship. I've managed to barricade myself in an escape pod."` 
```

> 			`Returning to your ship after an uneventful dinner at the spaceport, your ship's communication screen blares an alarm as the words EMERGENCY DISTRESS SIGNAL flash in red. A terrified looking man in a suit appears on the screen. 'I don't have much time — I'm kidnapped. If you're receiving this message, I need you to destroy the ship. I've managed to barricade myself in an escape shuttle."` 

Author: Fixed

> 			`You hear banging and muffled shouting over the intercom. 'I'm a Syndicate employee - I can pay you! Help me captain, you're my only h-'... the message cuts out...`

Reviewer 4: ```suggestion
			`You hear banging and muffled shouting over the intercom. 'I'm a Syndicate employee - I can pay you! Help me captain, you're my only h-' The message cuts out.`
```

> 			`You hear banging and muffled shouting over the intercom. 'I'm a Syndicate employee - I can pay you! Help me captain, you're my only h-'... the message cuts out...`

Author: Fixed or already changed

			choice
				`	(Assist the mysterious man.)`

> 						goto assist

Reviewer 4: ```suggestion
					goto assist
```

> 				`	(Decline to assist the man.)`

Author: Fixed in commit

> 						decline

Reviewer 4: ```suggestion
					decline
```

			label assist

> 			`	You note that the distress signal came from a warship in Moktar and wonder why a Syndicate employee would be kidnapped by pirates. Anyway, time to get ready for a fight.`

Author: Fixed in commit

				accept
	on enter "Moktar"
		"reputation: Independent (Killable)" = -1000
		conversation

> 			`When you enter Moktar, your sensors detect a pirate Vanguard, but no sign of any escape shuttle. Before you can respond, your ship intercepts a hail, and it opens communication.`

Reviewer 4: ```suggestion
			`When you enter Moktar, your sensors detect a pirate Vanguard, but no sign of any escape pod. Before you can respond, your ship intercepts a hail, and it opens communication.`
```

> 			`When you enter Moktar, your sensors detect a pirate Vanguard, but no sign of any escape shuttle. Before you can respond, your ship intercepts a hail, and it opens communication.`

Author: Reworded to hopefully avoid a medical crisis

> 			`	The voice of the captain of the Golden Jewel shouts out towards you. "You were unwise to get involved in our business dealings. Hand over that pompous salesman James and I'll let you live."`

Author: It looks like the <npc> tag only works if the npc has spawned previously, so keeping it hard coded for now. I agree with the other fix.

			choice
				`	"Uhhh... hand over who?"`

> 						goto who

Reviewer 4: ```suggestion
					goto who
```

				`	"I don't negotiate with pirates!"`

> 						goto negotiate

Reviewer 4: ```suggestion
					goto negotiate
```

				`	"You can take him, but I don't know where he is."`

> 						goto who

Reviewer 4: ```suggestion
					goto who
```

			label who
			`	"Bah, you lying buffoon! I'll find him in your ship's wreckage!"`
				goto power
			label negotiate
			`	"Aye, then I won't negotiate with you!"`
				goto power
			label power
			`	The Vanguard powers up its weapons.`
	npc kill
		government "Independent (Killable)"
		personality heroic vindictive staying
			confusion 20
		system "Moktar"
		ship "Vanguard" "Golden Jewel"		
		conversation 

> 			`The Golden Jewel has been destroyed. You receive a hail from a small escape pod. "Thank you Captain, you've saved my life. I'm James, meet me on <destination> and I'll give you your reward"`

Reviewer 4: Possible confusion with the tutorial James?
```suggestion
			`The Golden Jewel has been destroyed. You receive a hail from a small escape pod. "Thank you Captain, you've saved my life. I'm James, meet me on <destination> and I'll give you your reward."`
```

> 			`The Golden Jewel has been destroyed. You receive a hail from a small escape pod. "Thank you Captain, you've saved my life. I'm James, meet me on <destination> and I'll give you your reward"`

Author: I'm not sure if I understand "It is standard to use instead of your reward," but I made the punctuation edits and reworded the reward phrase.

			choice

> 				`	"Sounds good"`

Reviewer 4: ```suggestion
				`	"Sounds good."`
```

					accept
				`	"What's the deal with you and that pirate?"`
					goto piratedeal
			label piratedeal
			`"That pirate was a potential investor with the firm I work for. We had a disagreement over terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to go to <destination>.`
	on complete
		"reputation: Independent (Killable)" = 10
		conversation

> 			`When you land on <destination>, James is waiting for you at the Spaceport. "Thank you, truly. I'm James Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits."`

Reviewer 4: ```suggestion
			`When you land on <destination>, James is waiting for you at the spaceport. "Thank you, truly. I'm James Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits."`
```

> 			`When you land on <destination>, James is waiting for you at the Spaceport. "Thank you, truly. I'm James Vendeur, Investor Relations Associate with Maralex Controls. Anyway, I've managed to scrounge up my savings. It's not much to someone like you, I'm afraid. Just over ninety thousand credits."`

Author: Fixed/moot

			`	However, I see you're clearly a captain of means. I'd have to pull a few strings, but what if instead I get you into a private placement of Maralex stock? To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits."`
			choice
				`	"Sure, I'll invest. What do I need to do?"`
					goto invest
				`	"Maralex Controls? I thought you were part of the Syndicate."`
					goto maralex
				`	"Private placement? Does that mean I own part of the Syndicate?"`
					goto placement

> 				`	"No thanks, I'll take the money please."`

Reviewer 6: ```suggestion
				`	"No thanks. I'll take the money, please."`
```

					goto payup
			label maralex

> 			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We design, manufacture, and integrate innovative micro-control solutions for high-performance commercial and military applications for vehicles traveling by air and space. Succinctly, we make the airframes of small planes and spacecraft. Would you like to invest?"`

Reviewer 6: ```suggestion
			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We design, manufacture, and integrate innovative micro-control solutions for high-performance commercial and military air and space vehicles. Succinctly, we make the airframes of small planes and spacecraft. Would you like to invest?"`
```

(too many instances of the word "for" in one sentence)

Author: I agree, you've improved the wording.

> 			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We develop innovate micro-control solutions for high-performance commercial and military applications for vehicles traveling by sea, air, and of course space. Succinctly, we primarily develop fighter engines and drag designs. Would you like to invest?"`

Reviewer 4: ```suggestion
			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We develop innovative micro-control solutions for high-performance commercial and military applications for vehicles traveling by sea, air, and of course space. Succinctly, we primarily develop fighter engines and drag designs. Would you like to invest?"`
```

> 			`	"Maralex Controls is a partially-owned subsidiary of Syndicated Systems that accepts some private outside investment. We develop innovate micro-control solutions for high-performance commercial and military applications for vehicles traveling by sea, air, and of course space. Succinctly, we primarily develop fighter engines and drag designs. Would you like to invest?"`

Author: Oof. Changed that whole mini-section to `We design, manufacture, and integrate innovative micro-control solutions for high-performance commercial and military applications for vehicles traveling by air and space. Succinctly, we make the airframes of small planes and spacecraft.` Hope that works better?

			choice
				`	"Sure, I'll invest. What do I need to do?"`
					goto invest
				`	"Nope, I'll stick with the 90k."`	
					goto payup
			label placement

> 			`	"Almost, you will become a part owner of a partially owned subsidiary of Syndicated Systems -- Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest with us?"`

Reviewer 4: ```suggestion
			`	"Almost. You will own a fraction of a partially owned subsidiary of Syndicated Systems -- Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest with us?"`
```

> 			`	"Almost, you will become a part owner of a partially owned subsidiary of Syndicated Systems -- Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest with us?"`

Reviewer 5: ```suggestion
			`	"Almost; you will become a part owner of a partially owned subsidiary of Syndicated Systems -- Maralex Controls. As a privately held business, there are few opportunities to invest, and even fewer people who are permitted to. It's quite an exclusive investment, with quite high returns, I might add. Would you like to invest in us?"`
```
Not sure if `--` should stay; probably a single one is enough here.

Author: Fixed/moot and I agree on the --

			choice
				`	"Sure, I'll invest. What do I need to do?"`
					goto invest
				`	"Nope, I'll stick with the 90k."`
					goto payup
			label invest

> 			`	"Excellent. To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents, send over the payment, and a few other administrative tasks. I'll need to get a few approvals for the stock sale and set up the paperwork so it won't be ready today, but feel free to come back to my office anytime to execute the purchase. Remember, you'll need to have at least 1 million credits for the stock purchase."`

Reviewer 6: ```suggestion
			`	"Excellent. To purchase 1 share of Maralex Controls I will need 1 million credits. Then you sign a few documents, send over the payment, and take care of a few other administrative tasks. I'll need to get approval for the stock sale and set up the paperwork so it won't be ready today, but feel free to come back to my office here anytime to execute the purchase. Remember, you'll need to have at least 1 million credits for the stock purchase."`
```

> 			`	"Excellent, to purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents, send over the payment, and a few other administrative tasks. I'll need to get a few approvals for the stock sale and set up the paperwork so it won't be ready today, but feel free to come back to my office anytime to execute the purchase. Remember, you'll need to have at least 1 million credits for the stock purchase."`

Reviewer 5: ```suggestion
			`	"Excellent. To purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents, send over the payment, and a few other administrative tasks. I'll need to get a few approvals for the stock sale and set up the paperwork so it won't be ready today, but feel free to come back to my office anytime to execute the purchase. Remember, you'll need to have at least 1 million credits for the stock purchase."`
```

				accept
			label payup
			`	"Erhm, yes, of course. Here you are, my savings of ninety-one thousand, nine hundred and fifty one credits. I remain in your debt, and if you still want to invest in us, I will be here."`	
			action
				payment 91951
		log `Howard Vendeur from Maralex Controls offered to let you buy stock for 1 million credits on Sunracer.`

> mission "Syndicate Business 2, first stock purchase"

Reviewer 4: ```suggestion
mission "Syndicate Business 2"
```

	name "Purchase Stock on Hephaestus"
	description "Howard from Maralex Controls is letting you buy a share of the company for 1 million credits"
	priority
	passengers 1

> 	source 
> 		"Sunracer"

Reviewer 4: ```suggestion
	source "Sunracer"
```

> 	destination Hephaestus

Reviewer 4: ```suggestion
	destination "Hephaestus"
```

	to offer
		has "Syndicate Business 1, start: done"
		credits > 1000000
	on offer
		conversation
			`James once again thanks you profusely for saving his life and excitedly tells you what an exclusive offer you've received to be able to purchase a share in Maralex Controls.`
			`	"To reiterate, to purchase 1 share of Maralex Controls I will need 1 million credits and for you to sign a few documents. Our historical returns are 20% per annum which we expect to continue into perpetuity, paid out to you as a daily dividend of 500 credits. Do you accept these terms?"
			choice
				`	"Sure, here's the 1 million credits."`
					goto doit
				`	"Nope, I won't become an investor."`	
					decline
			label doit

> 			`After you've signed numerous dotted lines James says,"Alright, we have your payment and signatures. All that's left is to submit the paperwork to our headquarters and conduct a biometric scan on <destination>. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`

Reviewer 6: ```suggestion
			`After you've signed numerous dotted lines, James says,"Alright, we have your payment and signatures. All that's left is to submit the paperwork to our headquarters and conduct a biometric scan at our secure facility on <destination>. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`
```

(to explain why you can't do the biometric scan anywhere on the entire planet)

				accept
	on accept
		payment -1000000
	on complete
		conversation

> 			`You land on <destination> and follow James' instructions to the biometric scan center. You hand in the paperwork, and after scanning your fingerprint, eyes, and getting a blood sample, James comes over and says "Excellent, only one thing left to do.` 

Reviewer 6: ```suggestion
			`You land on <destination> and follow James' instructions to the biometric scan center. You hand in the paperwork, and after scanning your fingerprint, eyes, and getting a blood sample, James comes over and says, "Excellent, only one thing left to do.` 
```

> 			`You land on <destination> and follow Howard' instructions to the biometric scan center. You hand in the first set of paperwork you filled, and after scanning your fingerprint, eyes, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign."` 

Reviewer 4: ```suggestion
			`You land on <destination> and follow Howard's instructions to the biometric scan center. You hand in the first set of paperwork you filled, and after scanning your fingerprint, eyes, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign."` 
```

Author: done

> 			`	New regulations dictate that you have a choice in investing in our full spectrum of activities, or use the alleged 'ethical investment' option to avoid funding more ambiguous activities such as strip mining, clearing endangered habitats, and doing business on worlds with alleged human rights violations.`

Reviewer 4: Shouldn't this choice have been discussed before all the paperwork was signed?

			`	The second option would of course reduce your total returns by roughly 40%, resulting in a daily dividend of 300 credits. So, would you like to invest in the full investment suite or in the ethical investment option?"`
			choice
				`	"Full investment suite." (500 credits daily)`
					goto high
				`	"I'd prefer the ethical investment option." (300 credits daily)`	
					goto low
			label high
			action
				"salary: Syndicate Business" = 500
			label low
			action
				"salary: Syndicate Business" = 300

> 			`"Excellent, as a part owner of Maralex Controls, you will now receive a daily dividend. I have some business to attend to while I'm at our headquarters, but it was a pleasure doing business with you, Captain."`

Reviewer 4: ```suggestion
			`"Excellent. As a part owner of Maralex Controls, you will now receive a daily dividend. I have some work to attend to while I'm at our headquarters, but it was a pleasure doing business with you, Captain."`
```
